src/hp/netcheck.rs

```diff
@@ -162,7 +161,6 @@ impl Client {
                 current_hair_tx: None,
             },
             skip_external_network: false,
-            udp_bind_addr: "0.0.0.0:0".parse().unwrap(),
```

Comment:
This is the only place this field was ever initialised, and always to the same.  It was then read back via some functions.  So those functions and this state is removed, which is the main change in this PR.